lib/mixlib/shellout/windows/core_ext.rb

```diff
@@ -396,6 +413,22 @@ def unload_user_profile(token, profile)
       true
     end
 
+    def create_environment_block(env_pointer, token, inherit)
```

Comment:
Can we get yard comments above all these methods that describe what the methods do, what the params are, and what they return.